ipfs-wormhole.sh

```diff
@@ -27,9 +27,8 @@ function checkdep() {
 # Generate Passwords, the default length is 40 characters
 
 function generate_password() {
-  set +o pipefail
-  tr </dev/urandom -dc A-Za-z0-9 | head -c${1:-40}
-  set -o pipefail
+  local temp_password=$(< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c${1:-40})
```

Comment:
The list of characters given to `tr` allows `-`, which is used as a field separator in the `tag` (`IPFSHASH-PASSWORD-BASED64FILENAME`) and conflicts with the `gpg` command line. The list of character I give is slightly different (I also removed the `_`). I also tend to prefer uppercase letters for variables. If you update the list and variable name, and quote your "echo" below, I'll be more than pleased to merge your PR :-) :+1:

Reply:
Ah, yeah, didn't see that in my rush to fix what I broke. Updated.